package.json

```diff
@@ -118,7 +118,7 @@
 		"request-promise-native": "1.0.5",
 		"reselect": "4.0.0",
 		"scrypt": "6.0.3",
-		"selfkey-ui": "https://github.com/SelfKeyFoundation/selfkey-ui.git#02727d26329f4fc7f90a5a757ce2a3fdb4ef3dfd",
+		"selfkey-ui": "https://github.com/SelfKeyFoundation/selfkey-ui.git#08d61c4d11c39cdd90374ac31f486a35e248c87b",
```

Comment:
@sk91 could you update with latest from master?

Reply:
updated